config/custom_items.txt

```diff
@@ -629,4 +629,12 @@ item_path: /obj/item/clothing/glasses/hud/health
 item_name: Retinal Health Scanner
 item_icon: vichud
 item_desc: A small medical scanner similar to the one used in medical HUDs. This one has been designed to connect to and display the information directly on the wearer's implanted retinal overlay. The device includes a simple projector for displaying the data for users with unaugmented eyes.
-}
\ No newline at end of file
+}
+{
+ckey: your_neighbor_dave
```

Comment:
ckeys are normalized, so the underscore does not exist in this context. Hence why Broseph Stylin turns into brosephstylin (example on line 543)